t/22-svirt.t

```diff
@@ -234,4 +236,479 @@ subtest 'Method backend::svirt::open_serial_console_via_ssh()' => sub {
+        $svirt->_init_ssh($ssh_creds_svirt);
+        $svirt->_init_xml();
+
+        subtest 'vmware create=1' => sub {
+            my $dev_id       = 'device_id_001';
+            my $vm_name      = $svirt->name;
+            my $exp_filename = $svirt->name . $dev_id . '.vmdk';
+            my $exp_fullpath = $vmware_openqa_datastore . $exp_filename;
+            $svirt->add_disk({create => 1, size => '66G', dev_id => $dev_id});
+            my $cmd = shift @last_ssh_commands;
+            is(defined($cmd), 1, 'Command was triggered');
+
+            like($cmd, qr@\( set -x; vmid=\$\(vim-cmd vmsvc/getallvms \| awk '/$vm_name/ \{ print \$1 \}'\);if \[ \$vmid \]; then vim-cmd vmsvc/power\.off \$vmid;vim-cmd vmsvc/destroy \$vmid;fi; sleep 10 \) 2>&1@, "Check delete and destroy");
+            $cmd = shift @last_ssh_commands;
+            like($cmd, qr/vmkfstools -v1 --deletevirtualdisk $exp_fullpath/, "Check name of image to be deleted");
+            $cmd = shift @last_ssh_commands;
+            like($cmd, qr/vmkfstools -v1 --createvirtualdisk 66G --diskformat thin $exp_fullpath/, "Check size");
+
+            my $target_nodelist = $svirt->{domainxml}->findnodes('domain/devices/disk[@type="file" and @device="disk"]/target[@dev="hd' . $dev_id . '" and @bus="ide"]');
+            is($target_nodelist->size, 1, 'Only one <target> with that dev_id exists');
+
+            my $target_node = $target_nodelist->shift;
+            is(defined($target_node), 1, '<target> node exists');
+            my $source_node = $target_node->parentNode->findnodes('source')->shift;
+            is($source_node->getAttribute('file'), '[my_vmware_datastore] openQA/' . $exp_filename, 'XML dokument contains correct file set');
+        };
+
+        # Check backing file
+        subtest 'vmware backingfile=1' => sub {
+            @last_ssh_commands = ();
+            my $dev_id            = 'dev_id_002';
+            my $original_filename = 'foo_file.vmdk';
+            my $exp_filename      = $svirt->name . $dev_id . '.vmdk';
+            set_var(VMWARE_NFS_DATASTORE => 'nfs');
+            my $exp_fullpath = $vmware_openqa_datastore . $exp_filename;
+            my $vm_name      = $svirt->name;
+            $svirt->add_disk({backingfile => 1, size => '77G', dev_id => $dev_id, file => $original_filename});
+
+            my $cmd = shift @last_ssh_commands;
+            like($cmd, qr@\( set -x; vmid=\$\(vim-cmd vmsvc/getallvms \| awk '/$vm_name/ \{ print \$1 \}'\);if \[ \$vmid \]; then vim-cmd vmsvc/power\.off \$vmid;vim-cmd vmsvc/destroy \$vmid;fi; sleep 10 \) 2>&1@, "Check delete and destroy");
+            $cmd = shift @last_ssh_commands;
+            like($cmd, qr/vmkfstools -v1 --deletevirtualdisk $exp_fullpath/, "Check name of image to be deleted");
+            $cmd = shift @last_ssh_commands;
+            like($cmd, qr%cp\s+/vmfs/volumes/nfs/hdd/$original_filename\s+$vmware_openqa_datastore*;%, "Copy hdd to $vmware_openqa_datastore");
```

Comment:
```suggestion
            like($cmd, qr%cp\s+/vmfs/volumes/nfs/hdd/$original_filename\s+$vmware_openqa_datastore\*;%, "Copy hdd to $vmware_openqa_datastore");
```

Reply:
hm...I guess it wouldn't match the end anymore, or ?